providers/terraform/resources/terraform.lr

```diff
@@ -167,6 +167,12 @@ terraform.plan {
   terraformVersion string
   // Resource changes
   resourceChanges() []terraform.plan.resourceChange
+  // Variables used to generate the Terraform plan
+  variables []terraform.plan.variable
+  // Indicates if the plan is applyable or not
+  applyable bool
+  // Indicates if the plan errored or not
```

Comment:
```suggestion
  // Whether the plan errored
```

Reply:
Thank you!! I was following patterns in other places within this file, maybe we need a massive update in a few other providers 😅